src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -914,6 +766,10 @@ class ModelSystem(System):
     """
     Model system used as an input for simulating the material.
 
+    This version stores the atomic positions at the top level in the quantity “positions”
```

Comment:
I don't think you need to say "this version", you can just be more direct "Atom positions are stored at the top...", etc.